src/styles/main.scss

```diff
@@ -0,0 +1,43 @@
+@import './utils/mixin';
+@import './utils/variables';
+
+.page {
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  width: ($cellSize * $weakSize) +6px;
+  padding-inline: $cellPading;
+
+  &__day {
+    width: $cellSize;
+    height: $cellSize;
+    background-color: $cellCollor;
```

Comment:
There is a typo in the variable `$cellCollor`; it should be `$cellColor`. Consistent and correct naming conventions are important for maintaining a clear and understandable codebase.